test/queries/ledger_test.go

```diff
@@ -460,6 +465,122 @@ func TestLedgerTxProof(t *testing.T) {
 	})
 }
 
+// Scenario:
+// HTTP GET "/ledger/tx/receipt/{txId}" gets transaction receipt
+// HTTP GET "/ledger/proof/tx/{blockId}?idx={idx}" gets proof for tx with index idx inside block blockId
+// HTTP GET "/ledger/proof/tx/{blockId}?idx={idx}" with invalid query params
```

Comment:
All the `positive` part is missing...

Invoke the `HTTP GET "/ledger/proof/tx/{blockId}?idx={idx}` api and verify the results as we do in the SDK